usage/jsgui/src/test/java/org/apache/brooklyn/rest/jsgui/BrooklynJavascriptGuiLauncher.java

```diff
@@ -49,7 +49,7 @@
     public static void main(String[] args) throws Exception {
         // NOTE: When running Brooklyn from an IDE (i.e. by launching BrooklynJavascriptGuiLauncher.main())
         // you will need to ensure that the working directory is set to the jsgui folder. For IntelliJ,
-        // set the 'Working directory' of the Run/Debug Configuration to $MODULE_DIR/../jsgui.
+        // set the 'Working directory' of the Run/Debug Configuration to $MODULE_DIR$/../jsgui.
```

Comment:
It's an IntelliJ specific thing: https://www.jetbrains.com/idea/help/path-variables.html